cmd/list_bundles.go

```diff
@@ -0,0 +1,88 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/opdev/opcap/internal/bundle"
+
+	"github.com/spf13/cobra"
+)
+
+var listBundlesFlags struct {
+	bundlesDir  string
+	bundlesRepo string
+}
+
+func listBundlesCmd() *cobra.Command {
+	cmd := cobra.Command{
+		Use:   "bundles",
+		Short: "List all bundles and versions",
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			if listBundlesFlags.bundlesDir != "" && listBundlesFlags.bundlesRepo != "" {
+				return fmt.Errorf("can not specify both --from-dir and --from-repo at the same time")
+			}
+			return nil
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			err := listBundles(cmd.Context(), cmd.OutOrStdout())
+			if err != nil {
+				return err
+			}
+			return nil
+		},
+	}
+
+	flags := cmd.Flags()
+	flags.StringVar(&listBundlesFlags.bundlesDir, "from-dir", "",
+		"specifies the source directory with bundles under the operators directory, can't be used with --from-repo")
+	flags.StringVar(&listBundlesFlags.bundlesRepo, "from-repo", "https://github.com/redhat-openshift-ecosystem/certified-operators.git",
+		"Git repository URL from where to download bundles, can't be used with --from-dir")
```

Comment:
These can be marked as mutually exclusive and automatically checked.

https://pkg.go.dev/github.com/spf13/cobra#Command.MarkFlagsMutuallyExclusive